doc/sphinx/proof-engine/tactics.rst

```diff
@@ -3265,51 +3265,66 @@ the conversion in hypotheses :n:`{+ @ident}`.
+   the constant to which :n:`@qualid` refers in the current goal and
+   then replaces it with its :math:`\beta`:math:`\iota`-normal form.
 
-.. exn:: @qualid does not denote an evaluable constant.
-   :undocumented:
+   .. exn:: @qualid does not denote an evaluable constant.
 
-.. tacv:: unfold @qualid in @ident
+      This error is frequent when trying to unfold something that has
+      defined as an inductive type (or constructor) and not as a
+      definition.
 
-   Replaces :n:`@qualid` in hypothesis :n:`@ident` with its definition
-   and replaces the hypothesis with its :math:`\beta`:math:`\iota` normal form.
+      .. example::
 
-.. tacv:: unfold {+, @qualid}
+         .. coqtop:: abort all fail
 
-   Replaces *simultaneously* :n:`{+, @qualid}` with their definitions and
-   replaces the current goal with its :math:`\beta`:math:`\iota` normal form.
+            Goal 0 <= 1.
+            unfold le.
 
-.. tacv:: unfold {+, @qualid at {+, @num }}
+   .. tacv:: unfold @qualid in @goal_occurrences
 
-   The lists :n:`{+, @num}` specify the occurrences of :n:`@qualid` to be
-   unfolded. Occurrences are located from left to right.
+      Replaces :n:`@qualid` in hypothesis (or hypotheses) designated
+      by :token:`goal_occurrences` with its definition and replaces
+      the hypothesis with its :math:`\beta`:math:`\iota` normal form.
 
-   .. exn:: Bad occurrence number of @qualid.
-      :undocumented:
+   .. tacv:: unfold {+, @qualid}
 
-   .. exn:: @qualid does not occur.
-      :undocumented:
+      Replaces *simultaneously* :n:`{+, @qualid}` with their
+      definitions and replaces the current goal with its
+      :math:`\beta`:math:`\iota` normal form.
 
-.. tacv:: unfold @string
+   .. tacv:: unfold {+, @qualid at @occurrences }
 
-   If :n:`@string` denotes the discriminating symbol of a notation (e.g. "+") or
-   an expression defining a notation (e.g. `"_ + _"`), and this notation refers to an unfoldable constant, then the
-   tactic unfolds it.
+      The list :token:`occurrences` specify the occurrences of
+      :n:`@qualid` to be unfolded. Occurrences are located from left
+      to right.
 
-.. tacv:: unfold @string%@ident
+      .. exn:: Bad occurrence number of @qualid.
+         :undocumented:
 
-   This is variant of :n:`unfold @string` where :n:`@string` gets its
-   interpretation from the scope bound to the delimiting key :token:`ident`
-   instead of its default interpretation (see :ref:`Localinterpretationrulesfornotations`).
+      .. exn:: @qualid does not occur.
+         :undocumented:
 
-.. tacv:: unfold {+, @qualid_or_string at {+, @num}}
+   .. tacv:: unfold @string
 
-   This is the most general form, where :n:`qualid_or_string` is either a
-   :n:`@qualid` or a :n:`@string` referring to a notation.
+      If :n:`@string` denotes the discriminating symbol of a notation
+      (e.g. "+") or an expression defining a notation (e.g. `"_ +
+      _"`), and this notation refers to an unfoldable constant, then
```

Comment:
"refers to" seems wrong.  Do you mean "denotes an application whose head symbol is"?

Reply:
That would be a lot of "denotes" :smile: Note that I didn't write any of this, so it is a bit strong to say what "I mean".

Reply:
Applied your suggestion nonetheless.